src/graphql/explorer.ts

```diff
@@ -32,7 +32,7 @@ export default class Explorer extends CommandBase {
     return this;
   };
 
-  withNearText = (nearTextObj: any) => {
+  withNearText = (nearTextObj: object) => {
```

Comment:
What is `object`? shouldn't be here `NearText`?

Reply:
Updated with interface for NearText args